CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+
+KOA is heavily involved in [Habitica](http://www.habitica.com/), and your Habitica API Key will be required. You can find [your Habitica API here](https://habitica.com/user/settings/api), just don't forget you'll need to log in.
+In addition to this, it requires your Habitica token, to be able to fetch the COTW information from the Habitica API.
+<br>_Another friendly reminder to keep API keys secret to you. Always remove them from your code, or add them to a .gitignore, before submitting your work._
+
+#### Ready to test your modifications to Horace?
+
+In your root directory, go ahead and run ```npm start``` and you're ready to go.
+
+#### We have a linter for NodeJS
+
+We use [eslinter](https://eslint.org/) for code checkups. To run the linter, go ahead and run ```npm run pretest```.
+
+    The types of contributions you’re looking for
+    Your roadmap or vision for the project
+    How contributors should (or should not) get in touch with you
+
+#### Questions, Comments, Concerns <a name = "questions"></a>
+
+- **General questions about KOA or Horace?** Head on over to the [KOA Discord Server](https://discordapp.com/invite/EYX7XGG) for any sort of discussion. If you're looking to discuss development or features, we'd recommend the #computer-science channel.
+- **Development questions** or code problems? Search up your question on Stack Overflow or find general forums if your questions are related specifically to code, but no harm in asking other #computer-science Sectorians for discussions.
+- **Check out the FAW** [below!](#faq)
+
+## Pull Request Guidelines <a name = "pullreqguide"></a>
+
+_Please adhere to the following guidelines._ We want to avoid duplicate issues, reports, features, etc. 
+
+1. Search the [Issue Tracker](https://github.com/Knights-Of-Academia/horace/issues) for any relevant threads to what you're looking to create a PR for.
+2. Keep it detailed. We want to know your vision, and the more you give us the better discussion we can have!
+3. Please make sure you have read the [license agreement](LICENSE.md). We're using the [MIT License](https://opensource.org/licenses/MIT)- it's only 3 paragraphs!  
+_The following will need to become more detailed over time_
+4. Make a new branch to fork your changes from
+5. Make sure to test your bot on a local/personal testing Discord server
+6. Make sure your branch is updated with the master branch before sending your pull request, in case other changes have been made
+7. Send your pull request to the [develop branch](https://github.com/Knights-Of-Academia/horace/tree/develop)
+8. Follow the [commit message guidelines](#cmtmsg)
+
+### Commit Message Guidelines <a name = "cmtmsg"></a>
+If we want this, but 
+
+### Merging a PR <a name = "mergepr"></a> _KOA Team Only_
+- Passes our CL (changelist) guidelines [here](TBW)
```

Comment:
Would need to write these out, but realized Cl means Continuous Integration so this may not be necessary.